pkg/operator/operator.go

```diff
@@ -1061,3 +1073,53 @@ func (o *Operator) waitForClusterUpgrade(appSlug string) error {
 		time.Sleep(5 * time.Second)
 	}
 }
+
+// notifyUpgradeSucceeded sends a metrics event to the api that the upgrade succeeded.
+func (o *Operator) notifyUpgradeSucceeded(ctx context.Context, kbClient kbclient.Client, ins *embeddedclusterv1beta1.Installation, appID string) error {
```

Comment:
this function name can be confused with application upgrades. i would suggest putting `cluster` in the name.
```suggestion
func (o *Operator) notifyClusterUpgradeSucceeded(ctx context.Context, kbClient kbclient.Client, ins *embeddedclusterv1beta1.Installation, appID string) error {
```